configs/centernet/centernet_tta.py

```diff
@@ -0,0 +1,39 @@
+tta_model = dict(
+    type='DetTTAModel',
```

Comment:
Check whether this follows the official implementation of CenterNet. (I thought it is different from the original one, which simply fuse the score of model predictions) If not, we should add comments to indicate that this is different from the TTA of official CenterNet